### PR TITLE
Qué dicen nuestros socios

### PR DESCRIPTION
**Pull request by JaqKent**
## Summary

Que dicen nuestros socios

## Screenshots

image

## Trello Card

https://trello.com/c/wzsInxQQ

### CODE REVIEW

File: src/components/MainTitle/styles.module.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 }
 .title {
   color: darken($gray, 25);
-  font-family: Roboto;
+  font-family: roboto;
```
Comment (nathan-hyan) [inline src/components/MainTitle/styles.module.scss]: "Roboto" as string

File: src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+@import "scss/variables/_mixins";
+* {
+  font-family: "Roboto";
+}
+
+.container {
+  @include flex(space-around, center, column);
+  border-radius: 10px;
+  text-align: center;
+  display: inline-flex;
```
Comment (nathan-hyan) [inline src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss]: the `flex` mixin already has the display... either remove the mixin or this `display` declaration

File: src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  margin: 20px;
+  padding: 25px;
+  background-color: $primary;
+}
+
+.text {
+  color: $white;
+  font-size: 15px;
+  line-height: 18px;
+  white-space: pre;
```
Comment (nathan-hyan) [inline src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss]: `pre` i think doesn't exist... verify this pls

File: src/screens/Home/components/AboutUs/i18n.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+i18next.addResources('en', 'Comments', {
+  title: 'What Our Partners Say',
+  one: 'text. ',
+  nameOne: '-John Doe',
+  two: 'text ',
+  nameTwo: '-John Doe',
+  three: 'text.',
+  nameThree: '-John Doe'
+});
```
Comment (nathan-hyan) [inline src/screens/Home/components/AboutUs/i18n.ts]: Put either the same lorem ipsum text or some other stuff to fill this thing correctly.

Also, the `-` on the `John Doe`s has to be on the component itself, not on the constant. So you can only fill the name in here without the dash

File: src/screens/Home/components/AboutUs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+function AboutUs() {
+  const { t } = useTranslation('Comments');
+  return (
+    <div className={styles.container}>
+      <MainTitle text={t('title')} />
+
+      <div className={styles.boxContainer}>
+        {COMMENTS.map((comment) => (
+          <CommentBox key={comment.id} text={t(comment.text)} personName={t(comment.name)} />
```
Comment (nathan-hyan) [inline src/screens/Home/components/AboutUs/index.tsx]: `personName` must receive the text without translating it

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Some comments

File: src/components/MainTitle/styles.module.scss (hunk 1 above)
Comment (JaqKent) [reply src/components/MainTitle/styles.module.scss]: done

File: src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss (hunk 2 above)
Comment (JaqKent) [reply src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss]: done

File: src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss (hunk 3 above)
Comment (JaqKent) [reply src/screens/Home/components/AboutUs/components/CommentBox/styles.module.scss]: pre

File: src/screens/Home/components/AboutUs/i18n.ts (hunk 4 above)
Comment (JaqKent) [reply src/screens/Home/components/AboutUs/i18n.ts]: done

File: src/screens/Home/components/AboutUs/index.tsx (hunk 5 above)
Comment (JaqKent) [reply src/screens/Home/components/AboutUs/index.tsx]: done

**Review by nathan-hyan — APPROVED** [review]
🥇

**Pull request merged**